app/views/shared/_header.html.haml

```diff
@@ -1,4 +1,10 @@
 - unless Rails.env.development?
   :javascript
     window.onbeforeunload = function() { return true };
-= image_tag logo_image, style: 'width:20%; margin: auto; display: block;', class: 'right-align'
+.row
+  .col.s12
+    - if Rails.application.config.settings.use_company_logo
```

Comment:
we zouden dit ook impliciet kunnen maken;
``` ruby
if Rails.application.config.settings.use_company_logo.present?
```
Of denk je dat dat brakker is? Het gaat nu namelijk stuk als je geen logo hebt (sowieso btw, daar moeten we eigenlijk ook even op checken).

Reply:
Ja het is nu wat dubbelop. Ik pas het aan.